client/app/components/_pickupList/pickupListItem/pickupListItem.styl

```diff
@@ -0,0 +1,54 @@
+a
+    color: inherit
+    text-decoration: none
+    margin-left: .5em
+
+img
+    width: 2em
+    height: 2em
+    border-radius: 50%
```

Comment:
Ah, okay. Well, till then I'll just prefix the used classes with the component name then...